src/styles/calendar.scss

```
@@ -0,0 +1,40 @@
+@import "./variables.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding-inline: $padding-calendar;
+  gap: $gap;
+  width: $day-size * 7 + $gap * 6 + $padding-calendar * 2;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: #eee;
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid #000;
+    transition: transform 0.5s ease;
+
+    &:hover {
+      transform: translateY(-20px);
+      cursor: pointer;
+      background-color: #ffc0cb;
+    }
+
+    @for $day-num from 1 through $days {
+      &:nth-child(#{$day-num})::before {
+        content: "#{$day-num}";
+      }
+    }
+  }
+
+  @each $day, $margin in $days-of-week {
+    --start-day--#{$day}
+    :first-child {
```

```suggestion
    --start-day--#{$day} :first-child {
```
Could you try to use this syntax? Because I see your code doesn't include these generated selectors. So, I think there is a problem.